docs/tutorials/platform_beginner/6_embed_content.md

```diff
@@ -0,0 +1,184 @@
+# Step 6 - Embed content
+
+## Create the Landmark Content Type
+
+Now you need to create the second Content Type needed in the site, **Landmark**.
+
+Go to Admin Panel &gt; Content types, and under the "Content" group, create the Landmark Content Type.
+
+A Landmark is an interesting place that Rides go through. Each Ride may be related to multiple Landmarks.
+
+- **Name**: Landmark
+- **Identifier**: landmark
+
+Then create all Fields with the following information: 
+
+| Field Type   | Name             | Identifier       |  Required | Searchable | Translatable |
+| ------------ | ---------------- | ---------------- | --------- | ---------- | ------------ |
+| Text line    | Name             | `name`           | yes       | yes        | yes          |
+| Rich Text    | Description      | `description`    | no        | yes        | yes          |
+| Image        | Photo            | `photo`          | yes       | no         | no           |
+| Map location | Location         | `location`       | yes       | yes        | no           |
+
+Confirm the creation of the Content Type creation form.
+
+Create a "Landmarks" Folder add some Landmarks to it.
+Note that you will need pictures (for the Photo Field) to represent them.
+
+## Add Landmarks to Ride Content Type definition
+
+Now edit the Ride Content Type in order to add a Content Relation Multiple between the two Content Types.
```

Comment:
`Content Relation Multiple` or Multiple Content Relation if it's not an official name